en/config_mc/position_mode_smooth_vel.md

```diff
@@ -0,0 +1,65 @@
+# Jerk-limited Modes
+
+This type of setpoints generator is active in:
+- [Position mode](../flight_modes/position_mc.md) when [MPC_POS_MODE=3](../advanced_config/parameter_reference.md#MPC_POS_MODE) (_Smooth position control (Velocity)_).
+- Auto modes when [MPC_AUTO_MODE=1](../advanced_config/parameter_reference.md#MPC_AUTO_MODE) (_Jerk-limited trajectory_).
```

Comment:
You say auto modes - do you mean just "Mission mode" or will this also affect landing, takeoff, return modes?